.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,6 +4,8 @@
 
 *Testing (if applicable):*
 
+*Documentation added (if applicable):*
```

Comment:
Is there a way to mention that if it's not part of the PR to create an issue that tracks it?
```suggestion
*Documentation added/planned (if applicable):*
```